### DISCUSSION TITLE
Game crashing on start

### DISCUSSION THREAD

**Discussion by SayuriGabriela** [Draylar/identity#354] [Q&A]
i've installed the mod to my fabric 1.18.2 but it crashes when i open the game. I really don't know what i've done wrong

**Comment by James103** [comment]
Do you have the crash report? If so, can you please post it here?

**Reply by SayuriGabriela** [reply]
it just crash the game with this message: 
![image](https://user-images.githubusercontent.com/84648753/170831531-f3388e33-d192-4240-b424-374956c6f75b.png)

**Reply by James103** [reply]
Can you please send your `latest.log` along with any previous logs in your instance's `.minecraft/logs` folder?

**Comment by Shadowysn** [comment]
Did you install the [Architectury API](https://www.curseforge.com/minecraft/mc-mods/architectury-api) mod?
Identity 1.18.2 won't work without it.

**Reply by SayuriGabriela** [reply]
yes i've downloaded it but it still crash
